packages/cli/src/index.js

```diff
@@ -215,10 +215,20 @@ async function runYargs() {
   await loadPlugins(yarg)
 
   // Run
-  await yarg.parse(process.argv.slice(2), {}, (_err, _argv, output) => {
+  await yarg.parse(process.argv.slice(2), {}, (err, _argv, output) => {
+    // Configuring yargs with `strict` makes it error on unknown args;
+    // here we're propagating the exit code.
+    if (err) {
+      process.exitCode = 1
```

Comment:
If we're propagating the exit code (like the comment says), does it exist on `err`? If so, can we read it from there instead of always setting it to 1?

Reply:
not sure but i'll look into it

Reply:
as far as i can tell there's no `err.code` property; this seems to be errors yargs throws, not like errors from subprocesses or anything